### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Adamx1973** [RedGreenBlue09/WFAv7_Installer#46] [General]
hi, on Lumia 650 when in boot selector after pressing vol up key boot selector "restarts" and nothing happens bootloader etc unlocked

**Comment by RedGreenBlue09** [comment]
What is your windows 10 mobile version before installing? Newer versions won't work, so to be sure, try with the one that comes with the ffu.

**Reply by Adamx1973** [reply]
it was lastest😄 (1709) proabobly thats why then.
thanks🙂
